automx2/generators/outlook.py

```diff
@@ -29,6 +29,8 @@
 from automx2.ldap import STATUS_SUCCESS
 from automx2.model import Domain
 from automx2.model import Server
+from automx2.util import expand_placeholders
+from automx2.generators.apple import _sanitise
```

Comment:
Nope :D

Reply:
All methods starting with underscore `_`ar meant to be "private" methods for that class